analytical/utils.py

```diff
@@ -71,7 +83,7 @@ def get_identity(context, prefix=None, identity_func=None, user=None):
         try:
             if user is None:
                 user = get_user_from_context(context)
-            if user.is_authenticated():
+            if get_user_is_authenticated(user):
                 if identity_func is not None:
```

Comment:
I know this is not part of the changes of this PR, but ...

Simply `if identity_func:` would be more pythonic.

Reply:
Are you sure? `is not None` is more explicit I think?